servicemodules/discordConstants.py

```diff
@@ -0,0 +1,66 @@
+from config import ENVIRONMENT_TYPE
+from typing import Tuple
+
+#################################
+# RAW DATA FOR ALL CHANNEL IDS
+#################################
+
+welcome_channel_id: int = {"production": 743842679741481051, "development": 817518666349412352}[ENVIRONMENT_TYPE]
```

Comment:
I've seen people who prefer to use their own Discord server for development and in the future even production deployments might support more than one 😅 

We could have 1 file per server loaded dynamically somehow `import_module(f'discord-constants-{ENVIRONMENT_TYPE}.py')` or some JSON/YAML config file, that way it will be easier to support more servers, people would just create 1 more file and update their `.env`

But that sounds like a bigger change, I'm happy to merge this for now and think about more customization later.

Reply:
yes, i did consider something like having a class to represent the discord server, with all the id's as fields. But that wouldnt make it less tedious to define a new environment than just adding , "personal": xxxxx, to each line. The one downside of this i could find was that if we end up having 5 different environment types, this will look incredibly cramped, but it seems unlikely to me that that will be the case.

Reply:
git ignoring a file you don't want to share is much easier than `"personal": xxxx` on each line 😅

Reply:
aah, yea i had not factored in that that would get rid of awkward git handling, smart.
i have plans to build some tests that make sure that all these variables actually agree with the current layout of the server, so when i get on that ill see about making such a system as well